src/lib/text-field/text-field.html

```diff
@@ -1,18 +1,22 @@
 <template>
-  <div class="forge-text-field__wrapper" part="root">
-    <div class="forge-text-field forge-field" part="container">
-      <div class="forge-text-field__leading-container" part="leading-container">
-        <slot name="leading"></slot>
-      </div>
-      <div class="forge-field__label-input-container" part="label-input-container">
-        <slot></slot>
-        <slot name="label"></slot>
-      </div>
-      <slot name="trailing"></slot>
-      <div class="forge-field__addon-end-container" part="addon-end-container">
-        <slot name="addon-end"></slot>
-      </div>
-    </div>
-    <slot name="helper-text"></slot>
-  </div>
+  <forge-field id="field">
+    <slot slot="label" name="label"></slot>
+    <slot slot="start" name="start"></slot>
+    <slot></slot>
+    <slot slot="end" name="end"></slot>
+    <slot slot="end" name="clear-button">
+      <forge-icon-button aria-label="Clear" tabindex="-1" dense>
+        <forge-icon name="clear"></forge-icon>
+      </forge-icon-button>
+      <forge-tooltip>Clear</forge-tooltip>
```

Comment:
I think we should add a slot here, and set the tooltip type to "label" to remove the `aria-label`. This will allow for consumers to provide their own labels for localization.
```html
<forge-icon-button tabindex="-1" dense>
  <forge-icon name="clear"></forge-icon>
</forge-icon-button>
<forge-tooltip type="label">
  <slot name="clear-tooltip">Clear</slo
</forge-tooltip>
```